source/getting-started/index.markdown

```diff
@@ -42,7 +42,7 @@ Follow this guide if you want to get started with Home Assistant easily, or if y
 We will need a few things to get started with installing Home Assistant. The latest Raspberry Pi model makes a good and affordable starting point for your home automation journey. Links below are linking to Amazon US. If you're not in the US, you should be able to find these items in web stores in your country.
 
 - [Raspberry Pi 3 Model B+](http://a.co/ak2SQor) + [Power Supply](https://www.raspberrypi.org/help/faqs/#powerReqs) (at least 2.5A)
-- [Micro SD Card](http://a.co/gslOydD). Get one that is Class 10 as they are more reliable. Size 32 GB or bigger recommended.
+- [Micro SD Card](http://a.co/gslOydD). Ideally get one that is [Application Class 2](https://www.sdcard.org/developers/overview/application/index.html) as they handle small I/O much more consistently than cards not optimized to host applications. Size 32 GB or bigger recommended.
```

Comment:
The link is not to an A2 card though? This would be https://www.amazon.com/SanDisk-128GB-Extreme-microSD-Adapter/dp/B07FCMBLV6/ref=sr_1_1

Reply:
That is the exact card I'm using now.

Reply:
This would be an A2 https://www.amazon.com/SanDisk-Extreme-UHS-I-128GB-Adapter/dp/B07G3H5RBT/ref=sr_1_1?keywords=sd+card+a2+32gb&qid=1561230798&s=gateway&sr=8-1

Reply:
I did not want to touch the link as I thought it was an affiliate link and the project actually profits from it.

Reply:
Yeah I will update the links to affiliate links after I merge this, just so it has my name on it and not your name from the merge. Just to avoid weird questions.